app/models/role.rb

```diff
@@ -178,6 +179,12 @@ def ignore_locking
     self
   end
 
+  def self.create_locked!(role_attrs = {})
+    role = Role.create! role_attrs.merge(:modify_locked => true)
```

Comment:
I think this is not really needed anymore since you could just 
```
Role.ignore_locking do
  Role.create! attrs
end
``` 
where you need to. This method doesn't actually cause the created role to be locked afaict.